src/test/java/racingcar/RacingTest.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RacingTest {
+
+    @Test
+    void 자동차_대수와_라운드_횟수_값을_받아_레이싱_객체를_생성한다() {
+        Racing racing = new Racing(5);
+        assertThat(racing).isNotNull();
+    }
+
+    @Test
+    void 라운드가_한_번_진행되면_Car_play가_반드시_호출된다() {
+        Racing racing = new Racing(5);
+        List<Car> cars = racing.playRound();
+        assertThat(cars.get(0).getPlayCount()).isEqualTo(1);
```

Comment:
```suggestion
        Car car = new Car();
        Racing racing = new Racing(List.of(car));
        racing.playRound();

        assertThat(car.getDistance()).isGreaterThanOrEqualTo(0);
```

값을 꺼내서 비교하다보면 객체지향적 패턴을 쉽게 무너 트립니다.
객체간의 비교할수 있는방법이 없을지 고민해보아요~

> 테스트를 위한 생성자는 추가하면 문제가 될까요?

Reply:
이게 배경일거같기도 한데요 :)